.github/workflows/backend_tests.yml

```
@@ -19,14 +19,14 @@ jobs:
     steps:
 
       - name: Checkout
-        uses: actions/checkout@1e31de5234b9f8995739874a8ce0492dc87873e2 # infered from @v4
+        uses: actions/checkout@1e31de5234b9f8995739874a8ce0492dc87873e2 # https://github.com/actions/checkout/releases/tag/v4.0.0
```

Dependabot supports various formats. The other option is this 


```suggestion
        uses: actions/checkout@1e31de5234b9f8995739874a8ce0492dc87873e2 # v4.0.0
```

I personally prefer the version I committed because it is a link that resolves instead of a link we need to form manually

---

That's because `# infered from @v4...` is not the right syntax. It would need to be something like `# v4+`